libraries/linters/tn_linter.py

```diff
@@ -1,15 +1,160 @@
 from __future__ import print_function, unicode_literals
 from libraries.linters.markdown_linter import MarkdownLinter
 
+from libraries.linters.snippet_comparison import snippet_comparison
+'''
+   tn_linter.py
+'''
+
+import re
+import httplib2
+
+
 
 class TnLinter(MarkdownLinter):
 
-    def lint(self):
-        """
-        Checks for issues with translationNotes
+	def lint(self):
+		"""
+		Checks for issues with translationNotes
+		Use self.log.warning("message") to log any issues.
+		self.source_dir is the directory of source files (.md)
+		:return boolean:
+		"""
+		self.httplib2_instance = httplib2.Http()
```

Comment:
Should probably use https://github.com/unfoldingWord-dev/tx-manager/blob/develop/libraries/general_tools/url_utils.py instead.